src/screens/Learning.js

```diff
@@ -62,21 +66,23 @@ export default ({
   const selectAnswerCallback = useCallback(
     item => {
       if (item.id === currentPhrase.id) {
-        // TODO add to learned
+        // TODO add to learned 
+        removeFromSeenPhrases(item)
       } else {
-        // TODO add to seen
+        if (seenPhrases.every(phrase => phrase?.id !== currentPhrase?.id)) {
+          addSeenPhrase(currentPhrase)
+        }
       }
-
       setDisableAllOptions(true);
-
       const answerOptionsWithSelected = answerOptions.map(phrase => {
-        return {...phrase, isSelected: phrase.id === item.id};
+        return { ...phrase, isSelected: phrase.id === item.id };
       });
 
       setAnswerOptions(answerOptionsWithSelected);
     },
     [currentPhrase, setDisableAllOptions, answerOptions],
```

Comment:
seenPhrases has to be added to dependency array
[currentPhrase, setDisableAllOptions, answerOptions, seenPhrases]